### ISSUE TITLE
frozen set with unique elements from a union of frozen sets

### ISSUE THREAD

**Issue by rmu75** [serge-sans-paille/frozen#103]
I wondered if there is an easy way to merge a number of `frozen::set`s into one and purge it of duplicate entries? I managed to do something like that with `std::arrays`, merging them with this https://wandbox.org/permlink/arkvF7HpZGr4LKOa hack, but it needs some additional ctors in `bits::carray` and further trickery (it seems the result of `std::unique` is not a constant expression).

Surely I can't be the only one who wants to create a static string pool like that?

**Comment by serge-sans-paille** [comment]
Yeah, the issue is « how do I make the elements in the sequence unique, and reflect that in the type ». I've been giving it a try but fails, so I'd say there's no easy way, but I'll happily take any way to merge two std::array with unique elements into an std::array of unique elements

**Comment by rmu75** [comment]
My proof of concept: 
https://gist.github.com/rmu75/428c302a7bacc4b5e6ca622a70bfdeff

tested with clang 10 with --std=c++2a
